public/js/attendance-scripts/attendance-form-add-new-person.js

```diff
@@ -48,6 +52,10 @@ function jsValidateTable() {
 
 }
 
+//input: boolean - success; string - errorMessage
+//output: message box div element
+//description: creates an element that is used in feedback to the user
+// on whether or not adding a person was successful and what went right/wrong
```

Comment:
I'd suggest doing comments in the JSDoc style <http://usejsdoc.org/about-getting-started.html>. If you've ever done JavaDoc comment it is basically the same. So this would just be:
```js
/**
 * creates an element that is used in feedback to the user
 * on whether or not adding a person was successful and what went right/wrong
 *
 * @param {boolean} success - [some description]
 * @param {string} errorMessage - [some description]
 * @returns {type} message box div element
 */
```